storybookjs/storybook
# add vue storybook boilerplate

Issue: Storybook for Vue  with Vuetify Material Component Framework takes some time to setup. Also new users to Vue may not understand how to wrap a layout around a component (which isn't addressed in the current docs).

## What I did
Created a boilerplate Storybook-for-Vue project using Vuetify Material Component Framework. Having something like this would have helped me get started much faster. So I'm sharing for others. Includes stories for a component with two states and a layout which wraps the component.

## How to test
demo deployed to GitPages 
https://white-rabbit-japan.github.io/vuetify-storyboard-boilerplate/

Is this testable with jest or storyshots?
I don't think so.

Does this need a new example in the kitchen sink apps?
Example exists in GitPages if you want to include. 

Does this need an update to the documentation?
Probably not. But I did find some issues in the docs which I'll pull request separately.

If your answer is yes to any of these, please make sure to include it in your PR.
